File: src/C/1002.c
/*
 * 1002 - Area of a Circle
 * Author: Otavio
 * Date: 25/04/2024
 */
#include <stdio.h>
#define PI 3.14159

int
main()
{
    double R;
    scanf("%lf", &R);
    
    double A=PI*R*R;
    printf("A=%.4lf\n", A);
    return 0;
}

File: src/JS/1002.js
/*
 * 1002 - Area of a Circle
 * Author: Otavio
 * Date: 31/08/2024
 */
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const PI = 3.14159;
const R = parseFloat(lines[0]);

const A = PI * (R*R);

console.log('A='+A.toFixed(4));
